symphony-bdk-core/src/test/java/com/symphony/bdk/core/service/datafeed/impl/OnDiskDatafeedIdRepositoryTest.java

```diff
@@ -0,0 +1,84 @@
+package com.symphony.bdk.core.service.datafeed.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.symphony.bdk.core.config.BdkConfigLoader;
+import com.symphony.bdk.core.config.BdkConfigLoaderTest;
+import com.symphony.bdk.core.config.exception.BdkConfigException;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.config.model.BdkDatafeedConfig;
+import com.symphony.bdk.core.service.datafeed.DatafeedIdRepository;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+/**
+ * Test class for the {@link OnDiskDatafeedIdRepository}.
```

Comment:
The comment here is not needed IMHO

Reply:
I like it (personally), quite convenient when navigating through the code. You can easily jump to the tested class by CTRL+Click on the link. Could be optional, I agree.

Reply:
command + shit + t :)

Reply:
![Screenshot 2020-08-28 at 18 56 54](https://user-images.githubusercontent.com/39826516/91593343-41e6d300-e960-11ea-93d8-36ff71e9294b.png)

Did not know this one !